pkgs/by-name/ky/kyverno-chainsaw/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kyverno-chainsaw";
+  version = "0.1.7";
+  cli = "chainsaw";
+
+  src = fetchFromGitHub {
+    owner = "kyverno";
+    repo = "chainsaw";
+    rev = "v${version}";
+    hash = "sha256-v71qAJSpnbHd+jkPkR34IVNvwWLhm04TrPzduB6ZOgA=";
+  };
+
+  vendorHash = "sha256-lHV5Ik/L/Svn9AvcKZupq778ektEsbxfOkCts4Ocx9g=";
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildVersion=v${version}"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildHash=${version}"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildTime=1970-01-01_00:00:00"
+  ];
+
+  meta = with lib; {
```

Comment:
```suggestion
  ];
  
  doCheck = false; # requires running kubernetes
  
  meta = with lib; {
```

Reply:
is it working with doCheck = false ? 
i had asked chainsaw maintainer for above error [look](https://github.com/kyverno/chainsaw/issues/962#issuecomment-1972570190) we shall wait for that

Reply:
The `kyverno-chainsaw` package can be installed when `doCheck = false` is set, because the tests are **not** run, so that is "working". But the tests themselves are still broken within nixpkgs are not not "working."

Reply:
To clarify in [this](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/cluster/kyverno/default.nix) kyverno nix package the maintainer not used "doCheck = false" it also requires kubernetes cluster . i think its not related to require kubernetes cluster

Reply:
chainsaw mentor said unit test requires cluster so disable them